listenbrainz/webserver/static/js/src/components/Card.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+
+type CardProps = {
+  style?: React.CSSProperties;
+  className?: string;
+};
+
+export default class Card extends React.Component<CardProps> {
```

Comment:
is it hard to make this a function instead of a class?

Reply:
Not at all, I thought we would like to keep it a class component for consistency. I will convert it to a functional component.